repository_service_tuf/helpers/tuf.py

```diff
@@ -162,10 +160,10 @@ def _update_snapshot(
     targets = Targets()
     snapshot = Snapshot()
     timestamp = Timestamp()
-    root = Root()
 
     # Populate public key store, and define trusted signing keys and required
     # signature thresholds for each top-level role in 'root'.
+    roles = {}
```

Comment:
If we are going to use type annotations then it's a good practice for dictionaries to write their corresponding internal types (as you did for the `keys` list on line 182) or:
```suggestion
    roles: dict[str, Metadata[T]]` = {}
```
or if `Metadata[T]` is not accepted then just use `Metadata`.

Reply:
Good point!